src/build/__init__.py

```diff
@@ -13,23 +13,29 @@
 import difflib
 import logging
 import os
-import re
 import subprocess
 import sys
-import textwrap
-import types
 import warnings
 import zipfile
 
-from collections import OrderedDict
-from collections.abc import Iterator, Set
-from typing import Any, Callable, Mapping, MutableMapping, Optional, Sequence, Tuple, Type, Union
+from collections.abc import Iterator
+from typing import Any, Callable, Mapping, Optional, Sequence, TypeVar, Union
```

Comment:
Several of these can be imported from `collections.abc` - at least Callable, Mapping, Sequence, assumgin they are not used in a non annotation setting with parameters. Not a blocker.

Reply:
Okay.